components/eam/bld/config_files/definition.xml

```diff
@@ -348,12 +348,15 @@ Switch to enable or disable ECPP in CAM: 0=off, 1=on.
 <entry id="use_crm_cldfrac" valid_values="0,1" value="0">
 Switch to enable or disable fractional cloudiness in m2005 microphysics in CRM: 0=off, 1=on.
 </entry>
-<entry id="MMF_microphysics_scheme" valid_values="sam1mom,m2005" value="sam1mom">
+<entry id="MMF_microphysics_scheme" valid_values="sam1mom,m2005,p3" value="sam1mom">
```

Comment:
m2005 is not valid option. Don't you want to delete it?

Reply:
yea, I'll do that.

Reply:
done